packages/nuxt/src/app/composables/fetch.ts

```diff
@@ -105,14 +105,20 @@ export function useFetch<
     controller?.abort?.()
     controller = typeof AbortController !== 'undefined' ? new AbortController() : {} as AbortController
 
-    const isLocalFetch = typeof _request.value === 'string' && _request.value.startsWith('/')
```

Comment:
Maybe we could use `hasProtocol` from `ufo` to test `_request.value` instead?

Reply:
Good idea! I didn't think of that. Let me update it using `hasProtocol`. Thanks for suggesting that, @danielroe